src/commands/login.ts

```diff
@@ -88,7 +88,7 @@ export default class Login extends Command {
 
         accountName && this.log(`> Current account is: ${accountName}`);
 
-        return;
+        process.exit(0);
```

Comment:
Why do we need this?

Reply:
Use `process.exit()` to quickly stop the server. 
This can prevent delays in terminating the script and ensure faster access to the shell in the CLI.

Reply:
But then we're preventing Oclif from shutting down gracefully.